ceres_solver/package.xml

```diff
@@ -0,0 +1,41 @@
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<package>
+  <name>ceres_solver</name>
+  <version>1.11.0</version>
+  <description>ceres_solver</description>
+  <maintainer email="google-cartographer@googlegroups.com">Google Inc.</maintainer>
+  <license>New BSD</license>
+  <url type="website">http://ceres-solver.org/</url>
+  <author>Google Inc.</author>
+
+  <run_depend>catkin</run_depend>
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <build_depend>atlas</build_depend>
```

Comment:
Make these all "<depend>" since these are all not just needed during build time.